org.gnome.PasswordSafe.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "org.gnome.PasswordSafe",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gnome-passwordsafe",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
```

Comment:
Why does it require network access?

Reply:
Network access is needed when copying the stock database with Gio into a gvfs-goa drive. If this permission isn't set this will not work.

Reply:
That's the problem I'm talking about:
gi.repository.GLib.Error: g-io-error-quark: Error while getting peer-to-peer dbus connection: Could not connect: Connection refused (0)

Reply:
Removed the permission, access to gvfs-goa drives from within the flatpak is faulty anyway. I will take care of a proper fix/issue report later. Priority for now is to publish the app.